loom/common/platform/platformMobileIOS.mm

```diff
@@ -337,35 +369,64 @@ bool platform_wasOpenedViaRemoteNotification()
 ///checks if a given sensor is supported on this hardware
 bool platform_isSensorSupported(int sensor)
 {
-    ///TODO: 1844: Support sensors on iOS
-    return false;
+    BOOL supported = false;
+    
+    switch (sensor)
+    {
+        case 0:
```

Comment:
There isn't one. The only reference I found was here: https://github.com/LoomSDK/LoomSDK/blob/JG-Accelerometer/sdk/src/loom/Platform/Mobile.ls#L62